swatch-product-configuration/src/main/resources/subscription_configs/OpenShift/rhosak.yaml

```diff
@@ -20,31 +20,31 @@ serviceType: Kafka Cluster
 
 metrics:
   - id: redhat.com:rhosak:transfer_gb
-    rhm_metric_id: redhat.com:rhosak:transfer_gb
-    aws_dimension: transfer_gb
+    rhmMetricId: redhat.com:rhosak:transfer_gb
+    awsDimension: transfer_gb
     prometheus:
       queryParams:
         product: rhosak
         metric: kafka_id:haproxy_server_bytes_in_out_total:rate1h_gibibytes
         metadata_metric: subscription_labels
   - id: redhat.com:rhosak:cluster_hour
-    rhm_metric_id: redhat.com:rhosak:cluster_hour
-    aws_dimension: cluster_hour
+    rhmMetricId: redhat.com:rhosak:cluster_hour
+    awsDimension: cluster_hour
     prometheus:
       queryParams:
         product: rhosak
         metric: kafka_id:strimzi_resource_state:max_over_time1h
         metadata_metric: subscription_labels
   - id: redhat.com:rhosak:storage_gib_months
-    rhm_metric_id: redhat.com:rhosak:storage_gib_months
-    aws_dimension: storage_gb
+    rhmMetricId: redhat.com:rhosak:storage_gib_months
+    awsDimension: storage_gb
     prometheus:
       queryParams:
         product: rhosak
         metric: kafka_id:kafka_broker_quota_totalstorageusedbytes:max_over_time1h_gibibyte_months
         metadata_metric: subscription_labels
```

Comment:
I think that one needs to stay the way it is because it's part of the query params sent to the prometheus api?

https://github.com/RedHatInsights/rhsm-subscriptions/pull/2306/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R53

Reply:
Update - these don't directly get sent to the prometheus api.  They're used to inject values into the query template coming from the `src/main/resources/application-openshift-metering-worker.yaml`.

Updated the [SWATCH-1234 Refactor prometheus integration to use swatch-product-configuration](https://issues.redhat.com/browse/SWATCH-1234) card to call this out